### ISSUE TITLE
NPE when closing browser opened by IntelliJ + flutter web

### ISSUE THREAD

**Issue by mfichman** [flutter/flutter-intellij#3590]
## What happened

Closed browser opened by IntelliJ when using flutter web

## Version information

IntelliJ IDEA `2019.1.3` • Flutter plugin `io.flutter 36.1.4` • Dart plugin `191.7830`

Flutter 1.5.4-hotfix.2 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 7a4c33425d (7 weeks ago) • 2019-04-29 11:05:24 -0700
Engine • revision 52c7a1e849
Tools • Dart 2.3.0 (build 2.3.0-dev.0.5 a1668566e5)

## Exception

null

```
java.lang.NullPointerException
	at io.flutter.server.vmService.HeapMonitor$HeapObject.getAsJsonObject(HeapMonitor.java:55)
	at io.flutter.server.vmService.HeapMonitor$HeapObject.getEntries(HeapMonitor.java:59)
	at io.flutter.server.vmService.HeapMonitor$IsolateObject.getHeaps(HeapMonitor.java:92)
	at io.flutter.inspector.HeapState.handleIsolatesInfo(HeapState.java:104)
	at io.flutter.inspector.HeapDisplay$1.lambda$handleIsolatesInfo$0(HeapDisplay.java:64)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.desktop/java.awt.EventQueue$4.run(Unknown Source)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
	at java.desktop/java.awt.EventQueue.dispatchEvent(Unknown Source)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:729)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:678)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:373)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.desktop/java.awt.EventDispatchThread.run(Unknown Source)
```

**Comment by jacob314** [comment]
@terrylucas 
Surprising any of the heap monitor code is working with Flutter Web. We might want to avoid heap monitoring on flutter web.

**Comment by devoncarew** [comment]
We're seeing this one quite a bit.

**Comment by pq** [comment]
@terrylucas: if you aren't already, I'm happy to take a look...

**Comment by pq** [comment]
I have a pretty good bead on a fix but not able to reproduce the issue that prompts it.  @devoncarew: are you seeing this too?

**Comment by devoncarew** [comment]
I've not seen it myself; from this issue you need to be running flutter web? The other two reports of this (the closed issues) might have more repo clues.

**Comment by mfichman** [comment]
Yes, it only seems to happen when running flutter web!

**Comment by pq** [comment]
Thanks @mfichman.  This should be fixed w/ #3625 and available in the next release. 👍

**Comment by mfichman** [comment]
Awesome, thanks a million for the quick turnaround!
